style.css

```diff
@@ -0,0 +1,409 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,400;1,100;1,200&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
```

Comment:
As per the project requirements, you should use the same Font as in the provided design ie., `cocogoose` and `Lato`. Kindly use these two fonts in conjunction with the provided design because different elements in provided design might be using one of the two fonts mentioned above.